lib/main_common.pm

```diff
@@ -1696,7 +1701,9 @@ sub load_create_hdd_tests {
     loadtest 'console/hostname'       unless is_bridged_networking;
     loadtest 'console/force_cron_run' unless is_jeos;
     # Remove repos pointing to download.opensuse.org and add snaphot repo from o3
-    replace_opensuse_repos_tests if is_repo_replacement_required;
+    if (is_repo_replacement_required) {
+        replace_opensuse_repos_tests;
+    }
```

Comment:
Why did you change that? The inline is more common and I would like to keep the old approach. If you would change this back then LGTM